config.fish

```diff
@@ -4,22 +4,21 @@
 # Don't say anything!
 set fish_greeting
 
-# --files: List files that would be searched but do not search
-# --no-ignore: Do not respect .gitignore, etc...
-# --hidden: Search hidden files and folders
-# --follow: Follow symlinks
-# --glob: Additional conditions for search (in this case ignore everything in the .git/ folder)set FZF_DEFAULT_COMMAND='rg --files --no-ignore --hidden --follow --glob "!.git/*"'
-
 # Use nvim as the standard editor
 # -x stands for export (or make an environment var)
 set -x EDITOR 'nvim -f'
 
 # Use ripgrep with FZF to make sure that follow things and respect the ignore
 # - Check the ~/.ripgreprc file if you want to see what other things are set by default
+# --files: List files that would be searched but do not search
+# --no-ignore: Do not respect .gitignore, etc...
+# --hidden: Search hidden files and folders
+# --follow: Follow symlinks
+# --glob: Additional conditions for search (in this case ignore everything in the .git/ folder)set FZF_DEFAULT_COMMAND='rg --files --no-ignore --hidden --follow --glob "!.git/*"'
 set -x FZF_DEFAULT_COMMAND 'rg --files'
 
 # Use ASDF as the standard package manager for ruby, elixir, and node
-source ~/.asdf/asdf.fish
+# source ~/.asdf/asdf.fish
```

Comment:
I think asdf has this built in for mac. I'm not sure why I needed to comment it out. IT does still work